### PR TITLE
Adya rfm

### PR DESCRIPTION
**Pull request by mlippert**
Pull request created as a place to leave code comments. @DALove1025

### CODE REVIEW

File: src/css/brix.css [hunk 1]
```diff
@@ -1012,29 +1011,42 @@ div.brixQuestion.brixCodeQuestion .testCases td
 
 div.brixQuestion.brixCodeQuestion .codeQuestionTextarea textarea
 {
-    overflow: auto;
-    width: 90%;
-    height: 206px;
-    font-size: 11.375px;
+    overflow: hidden;
+    width: 250px;;
```
Comment (mlippert) [inline src/css/brix.css]: double `;` after the width. Also are we sure a fixed absolute pixel width is what we want?

File: src/css/brix.css [hunk 2]
```diff
@@ -1012,29 +1011,42 @@ div.brixQuestion.brixCodeQuestion .testCases td
...
+    font-size: 11px;
```
Comment (mlippert) [inline src/css/brix.css]: can we use relative font sizes instead of absolute?

File: src/css/brix.css [hunk 3]
```diff
@@ -1012,29 +1011,42 @@ div.brixQuestion.brixCodeQuestion .testCases td
...
+
+div.brixQuestion.brixCodeQuestion .codeQuestionTextarea .lineNums
 {
     position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 1px;
     width: 15px;
-    height: 200px;
+    font-size: 11px;
```
Comment (mlippert) [inline src/css/brix.css]: can we use relative font sizes instead of absolute?

File: src/css/brix.css (hunk 1 above)
Comment (emAmazed) [inline src/css/brix.css]: The width is now defined for brixCodeQuestion by relative size.

File: src/css/brix.css (hunk 2 above)
Comment (emAmazed) [inline src/css/brix.css]: Yes. fixed now.

File: src/css/brix.css (hunk 3 above)
Comment (emAmazed) [inline src/css/brix.css]: fixed with relative font size now. Thanks.

**Pull request closed without merging**